### ISSUE TITLE
R CMD check fails on `develop` branch (2020-07-01)

### ISSUE THREAD

**Issue by GegznaV** [cbeleites/hyperSpec#201]
[R CMD check](https://github.com/cbeleites/hyperSpec/actions/runs/154258256) fails on `develop` branch due to the following warning: 

![image](https://user-images.githubusercontent.com/12725868/86515943-76895300-be25-11ea-94bd-e280ad63cf4e.png)

**Comment by cbeleites** [comment]
That's essentially #161 I think

**Comment by GegznaV** [comment]
Were any functions removed from **hyperSpec** in the group of commits that  resulted in the R CMD check failure?

**Comment by cbeleites** [comment]
No, I think this was caused by the automatic  translation to markdown documentation, which translated a link to a topic into a link to a function. 

I removed it, since the link won't make sense in future with the separate file import packages.

**Comment by GegznaV** [comment]
Appveyor still fails with the message:

![image](https://user-images.githubusercontent.com/12725868/86539614-a6f4ee00-bf06-11ea-865e-f794167d7f45.png)

Wasn't Appveyor disabled for **hyperSpec**?

**Comment by GegznaV** [comment]
> No, I think this was caused by the automatic translation to markdown documentation

I fixed all warnings that were caused by the automatic translation. And after the modifications, R CMD check passed. Maybe some of the modifications were reverted?

I did not participate in the meeting several weeks ago when you discussed the workflow, but I think that it would be better not to commit to `develop` branch directly (the branch could be locked as `master` is) but only via pull requests as they are checked before merging. I think, it would be easier to track what was changed (as pull requests consist of a group of commits that have something in common/solve some larger issue) and to be sure that the changes result in passing R CMD check.

**Comment by cbeleites** [comment]
I tried to enable appveyor: in one of the video calls we decided that instead of telling people to `install_github()` which will be without documentation, we can point them to ready-to-install packages built by the CI. Which will also have the advantage that those packages will always have passed the checks.

However, I did not yet find the correct configuration settings for hyperSpec. Turns out, disabling appveyer needs the project removed on the appveyor web site - deleting the configuration is not sufficient.
